### DISCUSSION TITLE
Welcome to Medusa Discussions!

### DISCUSSION THREAD

**Discussion by ctlllll** [FasterDecoding/Medusa#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KaruroChori** [comment]
So many thumbs up to the original message (mine included) but no one talking :D.
My background is as a bio-technologist & information engineer.
I lack expertise to do research in this domain, but I like to integrate LLM and other generative models in my projects :).
I am currently in the process of validating Medusa for some applications I have in mind.
